<launch>
<!--	<node pkg="chassis_commu" type="chassis_commu_node" name="chassis_commu_node" required="true" output="screen" launch-prefix="xterm -e"> -->
	<node pkg="chassis_commu" type="chassis_commu_node" name="chassis_commu_node" required="true" output="screen" > 
		<param name="chassis_commu_port" type="string" value="/dev/chassisCOM4" />
		<param name="odom_frame" type="string" value="odom" />
		<param name="base_footprint_frame" type="string" value="odom_base_footprint" />
		<param name="chassis_commu_baudrate" type="int" value="115200" />
		<param name="chassis_commu_timeout" type="int" value="1000" />
		<param name="chassis_encoder_resolution" type="int" value="1200" />
		<param name="chassis_wheel_diameter" type="double" value="0.1288" />
		<param name="chassis_wheel_track" type="double" value="0.3559" />
		<param name="chassis_gear_reduction" type="double" value="1.0" />
	</node>
	<node pkg="tf" type="static_transform_publisher" name="odom_to_ekf_tf" args="0 0 0 0 0 0 1 ekf_combined odom 100" />
	
	<node pkg="tf" type="static_transform_publisher" name="front_left_sonar_tf" args="0.18 0.10 0.12 0 0 0.259 0.966 base_footprint front_left_sonar 100" />
	<node pkg="tf" type="static_transform_publisher" name="front_midd_sonar_tf" args="0.20 0 0.12 0 0 0 1 base_footprint front_midd_sonar 100" />
	<node pkg="tf" type="static_transform_publisher" name="front_right_sonar_tf" args="0.18 -0.10 0.12 0 0 -0.259 0.966 base_footprint front_right_sonar 100" />
	<node pkg="tf" type="static_transform_publisher" name="back_midd_sonar_tf" args="-0.20 0 0.12 0 0 1 0 base_footprint back_midd_sonar 100" />

	<node pkg="tf" type="static_transform_publisher" name="base_to_imu_tf" args="-0.12 0 0.30 0 0 1 0 base_footprint imu_link 300" />
</launch>
